webapp/Dockerfile

```diff
@@ -97,4 +105,4 @@ COPY --from=build ${DOCKER_WORKDIR}/locales ./locales
 COPY --from=build ${DOCKER_WORKDIR}/package.json ./package.json
```

Comment:
install for production

Reply:
Put an example `Dockerfile` into your documentation.

Reply:
Has to be `production=false` @Mogge . See above 👆🏼

I replace the `${DOCKER_WORKDIR}` here with `.` as well. @roschaefer 

The example Dockerfiles are in the rebranding repo https://github.com/Ocelot-Social-Community/Ocelot-Social-Deploy-Rebranding/tree/master/docker @roschaefer .